pkg/component/gardenerscheduler/gardener_scheduler.go

```diff
@@ -42,6 +49,22 @@ var TimeoutWaitForManagedResource = 5 * time.Minute
 
 // Values contains configuration values for the gardener-scheduler resources.
 type Values struct {
+	// ClientConnection holds values for the client connection.
+	ClientConnection ClientConnection
+	// LogLevel is the level/severity for the logs. Must be one of [info,debug,error].
+	LogLevel string
+	// SchedulerConfiguration provides the configuration for the SeedManager admission plugin.
+	Schedulers schedulerv1alpha1.SchedulerControllerConfiguration
```

Comment:
After the discussion in #8265, we don't need this either. We can simply apply some default configuration in `configmaps.go` and go with `MinimalDistance`.